manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
```

Comment:
I'm a bit concerned that we're specifying deps in two places. Is the idea to remove the install_third_party.py stuff and move it here? If not, what happens if there's a dep here that's not in install_third_party, and vice versa?